### DISCUSSION TITLE
What is the keybinding for focusing on  mainPanel?

### DISCUSSION THREAD

**Discussion by RHOOPH** [jesseduffield/lazygit#2910] [Q&A] [answered]
I can't find the shortcut to focus the main panel, i.e the panel that shows the information of the things that is currently focused/selected. 

It only scrolls using mouse wheels, How can I do it with keyboard?  

I can press enter and then it goes into the main panel but that is not what i'm looking for. 

Thanks in advance.

**Comment by mark2185** [comment, marked as answer]
I believe it's `Page up/down`.
